spec/plotting/dataframe_spec.rb

```diff
@@ -301,3 +301,86 @@ def == other
     it { expect { df.plot(type: :box, categorized: {by: :c, method: :color}) }.to raise_error ArgumentError }
   end
 end
+
+describe Daru::DataFrame, 'plotting dataframe using gruff' do
+  before { Daru.plotting_library = :gruff }
+  let(:df) do
+    Daru::DataFrame.new({
+      a: [1, 3, 5, 2, 5, 0],
+      b: [1, 5, 2, 5, 1, 0],
+      c: [1, 6, 7, 2, 6, 0]
+    }, index: 'a'..'f')
+  end
+
+  context 'bar' do
+    let(:plot) { instance_double 'Gruff::Bar' }
+    before { allow(Gruff::Bar).to receive(:new).and_return(plot) }
+    it 'plots bar graph' do
+      expect(plot).to receive :labels=
+      expect(plot).to receive(:data).exactly(3).times
+      df.plot type: :bar
+    end
+    
+    it 'plots bar graph with block' do
+      expect(plot).to receive :labels=
+      expect(plot).to receive(:data).exactly(3).times
+      expect(plot).to receive :title=
```

Comment:
Why are these specs necessary? From what I can see you're writing specs for Gruff here, not daru.